index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Blue Recycling</title>
+    <link rel="stylesheet" href="styling.css">
+</head>
+
+<body>
+    <div class="wrapper-sections">
+        <div class='section' id='newsletter'>
+            <div class="tablet">
+                <span id='join'>Join our</span>
+                <span id='news'>newsletter</span>
+                <p id='desktop-text'>Want to get the best and latest news from us? Fill in your email adress and we will
+                    update you with the
+                    latest news and how we will continue to strive for a better climate.</p>
+                <form>
+                    <input type="email" name="" id="">
+                    <button type="submit">Subscribe</button>
+                </form>
+            </div>
+            <p>Want to get the best and latest news from us? Fill in your email adress and we will
+                update you with the
+                latest news and how we will continue to strive for a better climate.</p>
+        </div>
+        <div class='section' id='about-us'>
+            <div id="tablet-about-us">
+                <span id='about'>About</span> <span id='us'>us</span>
+            </div>
+            <p>We at Blue Recycle love the enviroment and fashion. By choosing sustainable clothes and recycling your
+                worn
+                denim, you can help make a positive impact on the planet. The more we recycle, the more good we can do.
+            </p>
+            <div id='svg-div'>
```

Comment:
Also maybe a bit generic id with risk of being reused in other places. Could it be a class instead?